src/d3a/models/myco_matcher/__init__.py

```diff
@@ -1,23 +1,26 @@
+from d3a.d3a_core.util import is_external_matching_enabled
+from d3a.models.myco_matcher.external_matcher import ExternalMatcher
 from d3a_interface.constants_limits import ConstSettings
 
-from d3a.constants import BidOfferMatchAlgoEnum
 from d3a.models.myco_matcher.pay_as_bid import PayAsBidMatcher
 from d3a.models.myco_matcher.pay_as_clear import PayAsClearMatcher
 from d3a.d3a_core.exceptions import WrongMarketTypeException
+from d3a_interface.enums import BidOfferMatchAlgoEnum
 
 
 class MycoMatcher:
-    def __init__(self):
-        self.bid_offer_pairs = []
+    def init(self):
```

Comment:
What was the reason to make this a normal method, rather than using `__init__` as before? I think it would be good to document it in the docstring.

Reply:
This is because we need the MycoMatcher singleton at startup, but the matcher should be set later on